portfolio/src/main/java/com/google/sps/data/NameCommentPair.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class NameCommentPair {
```

Comment:
nit: perhaps just call this `Comment`, so that you can add other fields as-needed. And add a class javadoc noting that this is modeling the Comment entity.

Reply:
Got it! I also changed the comment field name to COMMENT_CONTENT to avoid confusion with the class being named Comment.